src/components/Popover/__tests__/Popover.test.tsx

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
```

Comment:
Can we add this tests via separate PR please? To check that current implementation pass them

Reply:
Yes

Reply:
https://github.com/yandex-cloud/uikit/pull/285